scripts/storage-facts.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+#this script generates storage (ZFS, iSCSI) facts out of the hostvars of each VM
```

Comment:
I think it would be a good idea to add to every function an example configuration otherwise its really tedious to read your source code other wise im ok with this